accounting/common/utils.py

```diff
@@ -18,25 +18,31 @@
 #
 
 """
-Management command for generating data (usually for a development environment) to play with.
+Invoice utilities.
```

Comment:
This is the worst I've ever seen of git making a guess at a file move. What I really did here was delete the management command which was unfinished, and later on add this `utils.py` file. Git thinks I just changed names.